bgpd/bgp_zebra.c

```diff
@@ -586,13 +585,14 @@ static int zebra_read_route(ZAPI_CALLBACK_ARGS)
 		if (add) {
 			inet_ntop(api.prefix.family, &nexthop, buf,
 				  sizeof(buf));
-			zlog_debug(
-				"Rx route ADD VRF %u %s[%d] %pFX nexthop %s (type %d if %u) metric %u distance %u tag %" ROUTE_TAG_PRI,
-				vrf_id, zebra_route_string(api.type),
-				api.instance, &api.prefix, buf, nhtype, ifindex,
-				api.metric, api.distance, api.tag);
+			zlog_debug("Rx route ADD VRF %s %s[%d] %pFX nexthop %s (type %d if %u) metric %u distance %u tag %" ROUTE_TAG_PRI,
+				   bgp->name_pretty,
```

Comment:
This would lead to the output of something like `VRF VRF default` (double `VRF`), maybe we can strip VRF then?

Reply:
I would agree

Reply:
fixed